### PR TITLE
Amirhossein-added homeworks

### PR DESCRIPTION
**Pull request by amirhshad**

### CODE REVIEW

File: HTML-CSS/homework/Amirhossein/week2/project/style.css [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+        text-align: center;
+    }
+
+}
+@media screen and (max-width: 600px) {
+    
+
+    #grid-floats .grid-item {
+<<<<<<< HEAD
+        display: flex;
```
Comment (n4cr) [inline HTML-CSS/homework/Amirhossein/week2/project/style.css]: You have a merge conflict. This should be resolved

File: HTML-CSS/homework/Amirhossein/week2/project/style.css [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+        height: auto;
+>>>>>>> 0d76a193efcf02392a190993dd1af7f64b920bf3
+        
+        
+        
+    }
+    
+    
+<<<<<<< HEAD
+    /* #grid-floats .grid-item:nth-child(1) {
```
Comment (n4cr) [inline HTML-CSS/homework/Amirhossein/week2/project/style.css]: Another conflict

File: HTML-CSS/homework/Amirhossein/week2/project/style.css [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+        flex-basis: 47%;
+        text-align: center;
+    }
+    
+    #grid-flex .grid-item:nth-child(1) img {
+        width: 60%;
+    }
+<<<<<<< HEAD
+     */
+=======
```
Comment (n4cr) [inline HTML-CSS/homework/Amirhossein/week2/project/style.css]: Conflict

**Review by n4cr — COMMENTED** [review]
You have merge conflicts. Goolge what it is and how it should be resolved

File: HTML-CSS/homework/Amirhossein/week2/project/style.css (hunk 1 above)
Comment (amirhshad) [reply HTML-CSS/homework/Amirhossein/week2/project/style.css]: I just resolved the conflict

**Pull request merged**